### DISCUSSION TITLE
Adding repeater capabilities

### DISCUSSION THREAD

**Discussion by agoliveira** [ExpressLRS/ExpressLRS#744] [Q&A]
I was wondering what would take to be able to create a wireless relay so the TX could be in a pole away from the radio.
If I understand correctly, the radio talks CSFRshot directly to the TX module, which I assume it's pure serial (TX/RS) in that case, just a serial/bluetooth dongle would do the trick, assuming it is able to handle the serial speed?

**Comment by mathewcommons** [comment]
I'm trying to do something similar, I was just going to wire the ppm pin from a 2.4ghz rx to the r9m module. I haven't tried it yet but I'd like it if the developers made it to where you could put a oled screen on it to monitor its status.

**Comment by koolakoff** [comment]
Repeater capabilities would be really useful for example in the mountains. Just build a repeater fpv drone with gps, put it high above you and control main drone via repeater..

Cons is that repeater drone will need 3 antennas - one to receive, one tx for forwarding, and one rx for itself

**Comment by MagysConsole** [comment]
Yes, it would be great to be able to use the TX module regardless of the hardware. For example, BetaFPV ELRS Micro TX has an XT30 for external power and a display with a joystick to control the module settings. Thus, it would be very convenient to place the module on an antenna tracker or antenna mast.
